mingus/containers/note.py

```diff
@@ -293,6 +295,11 @@ def __eq__(self, other):
         """Compare Notes for equality by comparing their note values."""
         if other is None:
             return False
+        
+        # added as was getting erros from play_Bar.set_key
+        if type(other) == str:
+            other = Note(other)
```

Comment:
Seems to me we should not compare Notes and strings by implicit conversion, but instead raise a TypeError to uncover bugs.